platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Telemetry.java

```diff
@@ -1,75 +1,157 @@
 package com.mapbox.mapboxsdk.maps;
```

Comment:
Same as before, if this is the implementation, could we move this file to a `modules` folder?

Reply:
This is not possible due to semver